src/diagnostics/default_diagnostics.jl

```diff
@@ -328,21 +328,16 @@ function default_diagnostics(
         snowyland_diagnostics = [
             "gpp",
             "ct",
-            "lai",
```

Comment:
I removed ones we never look at - since Gabriele said that adding diagnostics can slow things down